app/src/main/java/com/door43/translationstudio/newui/home/ImportDialog.java

```diff
@@ -232,6 +235,17 @@ private void importUri(Uri uri) {
         }
     }
 
+    private void applySourcesToProjects(Translator translator, String[] targetTranslationSlugs) {
```

Comment:
Thanks, I should have asked about the best place to put it.  I'm gathering that it is not a big performance hit to do this each time we open the project.